Documentation/network/standalone.rst

```diff
@@ -0,0 +1,1610 @@
+various use-cases and configuration examples.
+
+Quick-Start
+===========
+
+Given the Cilium Standalone Gateway can be operated outside of a Kubernetes
+cluster, this guide provides minimal Docker-based step-by-step instructions.
+
+As a one-time prerequisite, ensure that the BPF file system is mounted under
+``/sys/fs/bpf``. If this is not the case already, a mount unit file can be set
+up under ``/etc/systemd/system/sys-fs-bpf.mount`` ...
+
+.. parsed-literal::
+
+   [Unit]
+   Description=BPF mounts
+   DefaultDependencies=no
+   Before=local-fs.target umount.target
+   After=swap.target
+
+   [Mount]
+   What=bpffs
+   Where=/sys/fs/bpf
+   Type=bpf
+
+   [Install]
+   WantedBy=multi-user.target
+
+... and brought up through ``systemctl start sys-fs-bpf.mount``.
+
+.. note::
+
+  The above step is only required for distributions with older systemd versions.
+  For most modern distributions this step is not required as systemd will
+  automatically mount the BPF file system at its default location. See also
+  `GH-10955 <https://github.com/cilium/cilium/issues/10955>`_ for details.
+
+In this Quick-Start example, we enable the standalone layer 4 load-balancer
+functionality for both IPv4 and IPv6, with Maglev consistent hashing as the
+load-balancing algorithm. In our case, the networking device for the load-balancer
+is called ``bond0``. Multi-device setups are supported as well, and can be
+specified via ``--devices=bond0,eth0`` or through a device name regex.
+
+For the specified devices, we enable XDP acceleration for maximum efficiency.
+The vast majority of modern network drivers for 10/40/100G and beyond support
+XDP today. If you are unsure whether your network driver supports native XDP,
+then simply set ``--bpf-lb-acceleration=disabled`` instead and Cilium loads
+its programs into the tc BPF layer instead of XDP.
+
+.. parsed-literal::
+
+   docker run --name cilium-gateway -td \\
+     -v /sys/fs/bpf:/sys/fs/bpf \\
+     -v /lib/modules:/lib/modules \\
+     --privileged=true \\
+     --network=host \\
+     quay.io/cilium/cilium:|IMAGE_TAG| \\
+     cilium-agent \\
+     --enable-ipv4=true \\
+     --enable-ipv6=true \\
+     --datapath-mode=lb-only \\
+     --bpf-lb-algorithm=maglev \\
+     --bpf-lb-acceleration=native \\
+     --devices=bond0
+
+By default, the layer 4 load-balancer operates in Source NAT (SNAT) mode, meaning replies from
+backends will be returned to the load-balancer which will then perform reverse NAT and
+sends the reply back to the client. Advanced options such as Direct Server Return
+(DSR) are supported as well and described in later sections of this guide.
+
+Validate the Setup
+==================
+
+After deploying Cilium Standalone Gateway with the Quick-Start guide above, we can first
+validate that the Cilium agent is running in the desired mode:
+
+.. parsed-literal::
+
+   docker exec cilium-gateway cilium status --verbose
```

Comment:
This will syntax-highlight the commands correctly and avoid the sphinx literal substitution:

```suggestion
.. code-block:: shell-session

   $ docker exec cilium-gateway cilium status --verbose
```